### ISSUE TITLE
[QUESTION] HIGH CPU TEMP

### ISSUE THREAD

**Issue by vhtNov17** [Andrewthe13th/Inventory_Kamera#313]
> **NOTE**
> Please describe on each question as much as you can.

## What seems to be an issue / question?
im using a laptop with AMD Ryzen 5 4600H, 8.00 GB of RAM and running windows 10. everytime i scan my cpu temp rises from 85C to 100C. normally, my cpu temp stays around 45-60C (idling) and 80-85 while playing genshin. this issue have been around since the first version of the scanner. is this normal?

## What have you done / tried?
i tried amenoma scanner and no high temp issue at all

## Checklist before sending issue

- [x] I have checked Issue page and there is **none** similar/unrelated to mine.

**Comment by Cupcak3** [comment]
Yes this is 'normal'. There can be a lot of processing involved in using a computer to read text from an image. I am not sure how Amenoma processes images or what libraries are used. Inventory Kamera uses background threads to process images while the application continues to click through your inventory. I haven't looked much into seeing if anything can be done about temperatures.
